### DISCUSSION TITLE
lsblk parser is not really necessary the lsblk has `-J` option

### DISCUSSION THREAD

**Discussion by braindevices** [kellyjonbrazil/jc#296] [Ideas]
I found some of my colleagues actually using jc to parse lsblk output. Actually this is not necessary at all
> -J, --json
> Use JSON output format.  It's strongly recommended to use --output and also --tree if necessary.

This option is introduced [7 years ago](https://github.com/util-linux/util-linux/commit/4a102a4871fdb415f4de5af9ffb7a2fb8926b5d1). I am pretty sure almost all the distro has it already. I suggest we should deprecate this parser or at least add some warning to tell users to use `-J` directly.

**Comment by kellyjonbrazil** [comment]
I like the idea - the only thing is that it looks like Centos7 came with `lsblk` v2.23.2, which doesn't have this option. Since Centos7 is still running on lots of servers I think it might be premature to issue any deprecation warnings. I'm not against the idea, though.
